pkg/kubelet/kubelet.go

```diff
@@ -775,12 +777,14 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		}
 	}
 
+	eventBroadcaster := toolsevents.NewBroadcaster(&toolsevents.EventSinkImpl{Interface: klet.kubeClient.EventsV1beta1().Events("")})
```

Comment:
Instead of creating it here, this should be added as a field to kubeDeps (i.e. Dependencies struct)